web/webpack.config.js

```diff
@@ -83,4 +87,7 @@ module.exports = {
     path: `${__dirname}/public/dist`,
     pathinfo: false,
   },
+  cache: {
+    type: 'filesystem',
+  }
```

Comment:
We probably need to change the docker config to cache this file.
Default location is `node_modules/.cache/webpack` but we can change that if needed.

Reply:
Hmm, I'm unsure what consequences this have - can we start simple to get this PR in (as you suggested)? :)

Reply:
The only consequences it should have is that the first run (no cachefile) will be slower but it will speed up subsequent builds.

Since we use docker things get a little tricker though so I'll remove it from this PR and then we can evaluate it at a later date, perhaps after we remove the nodejs frontend server; After that it would be very simple to cache this file for the CI and for local development it wouldn't need any caching at all.

Reply:
I think that makes good sense, let's start small :D